### DISCUSSION TITLE
[2725260]ENDER MAGNOLIA: Bloom in the Mist

### DISCUSSION THREAD

**Discussion by Niguyt5874** [pjy612/SteamManifestCache#557] [Share]
`appid:2725260 `
keys:
```
2725261;935fccb2fcc5c04d8e1f15711fb86eeb0efc2bba42b9b452e559a9c053ab3d82
```
```
228989;ad69276eb476cf06c40312df7376d63deac0c838b9a2767005be8bb306ffb853
```
```
228990;44d8c45ce229a11c4f231a3d2a350eaf80b0d69a8af938ec7ccca720f694b0e8
```

[ENDER MAGNOLIA Bloom in the Mist.zip](https://github.com/pjy612/SteamManifestCache/files/14814289/ENDER.MAGNOLIA.Bloom.in.the.Mist.zip)

`appid: 2429860`
keys:
```
2429861;591782e366a2be7556b05c52c3ebab70bf328172f2277b1fea48b311ac3c8423
```
```
2429862;ca36c8d20f6726a752664b377430a930b5579b93916c6a88a475a62391f53425
```
```
2429863;90981bee3efdee85d4251d950a4becc37d8433d2e1f0228a8022ae3f4416dc23
```
[妹相随～黑白世界的缤纷冒险～.zip](https://github.com/pjy612/SteamManifestCache/files/14814383/default.zip)

`appid:2475040 `
keys:
```
2475041;7c995ab124aade52717a87ffc7682731449c5546d4922ebcba6232b6d0d92a99
```

[纯爱独白：小小 Purelove Monologue Xiaoxiao.zip](https://github.com/pjy612/SteamManifestCache/files/14814402/Purelove.Monologue.Xiaoxiao.zip)

`appid:2521180 `
keys:
```
2521181;0d08c1a541dc361816efb1d2aee187fd512befa60d7d838d31220110ac24d7ce
```

[迷走二律背反.zip](https://github.com/pjy612/SteamManifestCache/files/14814425/default.zip)

**Comment by pjy612** [comment]
符号用错了...

**Reply by Niguyt5874** [reply]
浏览器自动翻译符号,我复制模板也是错的,给我整不会了.

**Reply by pjy612** [reply]
有个清单没上传完

**Reply by Niguyt5874** [reply]
行了
